templates/container-instances.json

```diff
@@ -22,7 +22,8 @@
     },
     "command": {
       "type": "array",
-      "defaultValue": [],
+      "defaultValue": [
```

Comment:
Any particular reason for the formatting change on this? Ditto the instance below.